index.js

```diff
@@ -22,35 +22,43 @@ async function run() {
     const pylintDisable = core.getInput('pylint-disable') || "";
     const pylintRCFile = core.getInput('pylint-rcfile') || "";
     const pylintPaths = core.getInput('pylint-paths');
+    const pylintIgnorePaths = core.getInput('pylint-ignore-paths') || "";
 
     let pylintOutput = "";  // Pylint process output.
-    try {
-        // Run pylint via `exec()`, which throws an exception if the
-        // process exits with a non-zero exit code. Here, we assume a
-        // non-zero exit code (and hence exception) means pylint has
-        // detected issues in the code that we need to report.
-        await exec.exec("pylint", [
-            `--disable=${pylintDisable}`,
-            `--rcfile=${pylintRCFile}`,
-            `--output-format=json`,
-            ...pylintPaths.split(" ")
-        ], {
-            listeners: {
-                stdout: (data) => pylintOutput += data.toString()
-            }
-        });
-    } catch (pylintError) {
+
+    // Run pylint via `exec()`, which throws an exception if the
+    // process exits with a non-zero exit code.
+    await exec.exec("pylint", [
+        `--disable=${pylintDisable}`,
+        `--ignore-paths=${pylintIgnorePaths}`,
+        `--rcfile=${pylintRCFile}`,
+        `--output-format=json`,
+        ...pylintPaths.split(" ")
+    ], {
+        listeners: {
+            stdout: (data) => pylintOutput += data.toString()
+        }
+    }).catch(pylintError => {
```

Comment:
Ahh cool, I have to admit it's been a _long_ time since I've ben writing JS 😄 I had been looking at [the modules tests](https://github.com/actions/toolkit/blob/main/packages/exec/__tests__/exec.test.ts) that still seemed to use it, but I guess that's as the test functions do need to be async?

Reply:
In the linked case, they mixed new-style `await` with old-style callback chaining.  They still need to `await` because they have logic that must execute afterward (whereas we don't).  I guess the way they've done it is slightly fewer lines of code than using a `try`/`catch`, and perhaps more readable.

Regardless, we can skip this minor niggle, it doesn't do any harm.